config/passport.js

```diff
@@ -139,6 +138,30 @@ function handleOauthLogin(profileMapper){
 }
 
 /**
+ * Maps Facebook profile info to User.profile
+ * @param profile
+ * @returns {{id: *, name: *, location: *, picture: *}}
+ */
+function mapFacebookProfile(profile){
```

Comment:
Please use http://graph.facebook.com/67563683055/picture?type=square for getting profile picture. And also put in blank strings for fields we cannot get.